format/spec.md

```diff
@@ -576,12 +588,12 @@ Manifest list files store `manifest_file`, a struct with the following fields:
 
-| v1         | v2         | Field id, name          | Type          | Description |
-| ---------- | ---------- |-------------------------|---------------|-------------|
-| _required_ | _required_ | **`509 contains_null`** | `boolean`     | Whether the manifest contains at least one partition with a null value for the field |
-| _optional_ | _optional_ | **`518 contains_nan`**  | `boolean`     | Whether the manifest contains at least one partition with a NaN value for the field |
-| _optional_ | _optional_ | **`510 lower_bound`**   | `bytes`   [1] | Lower bound for the non-null, non-NaN values in the partition field, or null if all values are null or NaN [2] |
-| _optional_ | _optional_ | **`511 upper_bound`**   | `bytes`   [1] | Upper bound for the non-null, non-NaN values in the partition field, or null if all values are null or NaN [2] |
+| v1         | v2         | Field id, name          | Type          | Description                                                                                                                                                                                                                                                        |
+| ---------- | ---------- |-------------------------|---------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| _required_ | _required_ | **`509 contains_null`** | `boolean`     | Whether the manifest contains at least one partition with a null value for the field                                                                                                                                                                               |
+| _optional_ | _optional_ | **`518 contains_nan`**  | `boolean`     | Whether the manifest contains at least one partition with a NaN value for the field                                                                                                                                                                                |
+| _optional_ | _optional_ | **`510 lower_bound`**   | `bytes`   [1] | Lower bound for the non-null, non-NaN values in the partition field, or null if all values are null or NaN [2]. For Geometry type, this is a Point composed of the min value of each dimension among all non-null, non-NAN Geometry values in the partition field. |
+| _optional_ | _optional_ | **`511 upper_bound`**   | `bytes`   [1] | Upper bound for the non-null, non-NaN values in the partition field, or null if all values are null or NaN [2]. For Geometry type, this is a Point composed of the max value of each dimension among all non-null, non-NAN Geometry values in the partition field. |                                                                                                                                                  |
```

Comment:
I don't think that this could ever be a `geometry` because the only way that would happen is using the `identity` transform, which is probably not allowed.

Reply:
My bad, let me revert this one